lib/graphql/schema/printer.rb

```diff
@@ -132,16 +132,22 @@ def print_input_value(arg)
           def print_value(value, type)
             case type
             when FLOAT_TYPE
+              return 'null' if value.nil?
```

Comment:
Yeah, my worry was that if we early return `if value.nil?` you lose the type-checking for that `type`.

> I see that NonNullType doesn't have this early return

Correct, I did this because a `NonNullType` cannot be `null` but its elements can.